internal/git-remote-gittuf/ssh.go

```diff
@@ -541,7 +538,7 @@ func handleSSH(repo *repository.Repository, remoteName, url string) (map[string]
 			// TODO: gittuf verify-ref for each dstRef; abort if
 			// verification fails
 
-			if !rslPushed {
+			if !dstRefs.Has(rsl.Ref) {
```

Comment:
We'll have to do a bit more tweaking in ssh because the push command also specifies the dst which will potentially change for the RSL. I'll submit a follow up patch though.